src/main/java/com/patsi/service/RecipeService.java

```diff
@@ -29,37 +29,37 @@ public class RecipeService {
     @Value("${spring.web.resources.static-locations[3]}")
     private String IMAGE_PATH;
 
+    //Get Existing Recipe
+    public List<Recipe> getRecipe(String uid) {
+        List<Recipe> finalRecipeList = recipeRepository.findAll().stream()
```

Comment:
avoid findAll. Should only query recipe by uid.

Image there are million of recipe, the memory will explode

Reply:
Fixed